core/src/cf/mutations.rs

```diff
@@ -21,7 +22,7 @@ pub enum TableMutation {
 	Def(DefineTableStatement),
 	#[revision(start = 2)]
 	// Includes the previous value that may be None
-	SetPrevious(Thing, Value, Value),
+	SetWithDiff(Thing, Value, Vec<Operation>),
```

Comment:
This is allowed because the store is gated behind a feature flag. Nobody has this change yet.

Reply:
To clarify, you mean the whole `TableMutation` has been used only behind the feature flag?

Reply:
Just that entry

Reply:
My undestanding:

- We use `Set` today, and keep using `Set` without the feature flag.
- With the feature flag, we use `SetWithDiff` instead of `Set`.

Is that correct?!

Reply:
Yes exactly. Set is essentially being deprecated in favour of SetWithDiff

Reply:
Sur-337, closing